source/common/midi_creation.py

```diff
@@ -0,0 +1,37 @@
+from midiutil.MidiFile import MIDIFile
+import pygame
+
+def MakeSong(list):
+    #pitch, bpm, duration, volume, instrument, amount
+    amount_of_instruments = len(list)
+    for shape in list:
+
+        # create MIDIFile object
+        midi = MIDIFile(amount_of_instruments, removeDuplicates=False)
+
+        # add tracks
+        track1 = 0
+        time = 0
+        channel = 0 
+        instruments = 0
+
+        while instruments < amount_of_instruments:
+        # create ass many tracks as instruments        
+            midi.addTrackName(instruments, time, f"Track{instruments}")
+            midi.addTempo(instruments, time, shape.bpm)
+            midi.addProgramChange(instruments, 0, time, shape.instrument)
+            midi.addNote(track1, channel, shape.pitch, time, shape.duration, shape.volume)
+            time = +2
```

Comment:
Is it correct?  
- Did you expect it to increment (because this won't...  
- Did you mean to set this value to `time = 2` because then the + is redundant.

Reply:
This was just the basic code I got from Giel, havent looked into this part yet. It is just an example for creating sounds, it is adding 2 to the time each time it goes through the while loop